cmd/ketch/output/yaml.go

```diff
@@ -0,0 +1,24 @@
+package output
+
+import (
+	"fmt"
+	"io"
+
+	"sigs.k8s.io/yaml"
+)
+
+// YAML represents data and a writer for yaml output type
+type YAML struct {
```

Comment:
is there a chance to start with unexported types? 
`YAML`, `JSON` and `Column` are consumed by this package only

```
type yaml struct {
	Data   interface{}
	Writer io.Writer
}

func (y *yaml) Write() error {
	d, err := yaml.Marshal(y.Data)
	if err != nil {
		return err
	}
	_, err = fmt.Fprintln(y.Writer, string(d))
	return err
}
```

Reply:
Good call. I privatized everything except the one `Write()`.